examples/example-firewall/main.tf

```diff
@@ -3,5 +3,5 @@ module "vnet" {
     source = "github.com/garnertb/github-runner-vnet//modules/firewall"
 
     base_name = var.base_name
-    github_enterprise_id = var.github_enterprise_id
+    github_business_id = var.github_enterprise_id
```

Comment:
There's a few other places where similar changes need to be made, I've created another PR to handle these so let's roll these back here.